src/presentation/http/schema/ResponseSchema.ts

```diff
@@ -0,0 +1,11 @@
+import type { NotePublic } from '@domain/entities/notePublic.js';
+
+/**
+ * Schema for the note response
+ */
+export interface NoteResponse {
+  note: NotePublic;
+  accessRights: {
+    canEdit: boolean;
+  };
```

Comment:
It looks redundant and reduces readability of route-handler. And also it's not a schema, so it's better to remove it back.